tests/controlboard/ControlBoardTest.cc

```diff
@@ -0,0 +1,206 @@
+                auto jointWrench = joint.TransmittedWrench(_ecm).value().at(0).torque();
+                auto jointPosition = joint.Position(_ecm).value().at(0);
+                auto jointVelocity = joint.Velocity(_ecm).value().at(0);
+
+                double jointAcceleration{0};
+                if (std::abs(jointVelocity) < 1e-6 && std::abs(jointVelocityPreviousStep) < 1e-6)
+                {
+                    jointAcceleration = 0;
+                } else
+                {
+                    jointAcceleration = (jointVelocity - jointVelocityPreviousStep)
+                                        / (_info.dt.count() / static_cast<double>(1e9));
+                }
+                std::cerr << "joint axis: " << jointAxis[0] << ", " << jointAxis[1] << ", "
+                          << jointAxis[2] << ", position: " << jointPosition
+                          << ", joint velocity: " << jointVelocity
+                          << ", joint acc: " << jointAcceleration
+                          << ", joint torque: " << jointWrench.x() << ", " << jointWrench.y()
+                          << ", " << jointWrench.z() << std::endl;
+
+                // Get link kinematic quantities
+                auto parentLinkPose = parentLink.WorldPose(_ecm).value();
+                // Compute link pose in body frame
+                auto linkPoseBody = parentLinkPose.CoordPoseSolve(link.WorldPose(_ecm).value());
+
+                auto theta = linkPoseBody.Roll();
+                auto thetaDot = link.WorldAngularVelocity(_ecm).value().X();
+                auto thetaDDot = link.WorldAngularAcceleration(_ecm).value().X();
+                std::cerr << "theta: " << theta << ", theta_dot: " << thetaDot
+                          << ", theta_ddot: " << thetaDDot << std::endl;
+
+                // Check that link and joint quantities are equal
+                // ASSERT_NEAR(joint_position, theta, 1e-3);
+                ASSERT_NEAR(jointVelocity, thetaDot, acceptedTolerance);
+
+                // From dynamics 2nd law it is possible to compute the expected joint torque
+                auto expectedJointTorque
+                    = -linkMass * gravity.Z() * linkLength / 2.0 * std::sin(jointPosition)
+                      + linkInertiaAtLinkEnd * thetaDDot;
+
+                // Get joint torque from control board
+                double jointTorque{};
+                iTorqueControl->getTorque(0, &jointTorque);
+
+                std::cerr << "Torque measured: " << jointTorque
+                          << " - expected: " << expectedJointTorque << std::endl;
+
+                if (iterations > 1)
+                {
+                    EXPECT_NEAR(jointTorque, expectedJointTorque, acceptedTolerance);
+                }
+
+                iterations++;
+                jointVelocityPreviousStep = std::abs(jointVelocity) < 1e-6 ? 0 : jointVelocity;
+                std::cerr << "========== Test PostUpdate done" << std::endl;
```

Comment:
```suggestion
                std::cerr << "========== Test PostUpdate done" << std::endl;
```

Not sure if you want review all this output messages, not sure what could be removed.

Reply:
Maybe until the issue we're having on this test is not solved I would leave them.